<style	type="text/css">
	B 	{
		font-family:helvetica;
		color:#404040;

		}
	body	{
			font-family:helvetica;
			color:#404040;
			font-size:105%;
			line-height:1.3;
			}
	blockquote	{
				margin-left:4.0em;
				margin-right:15.0em;
				}
	p	{
		font-family:helvetica;
		color:#404040;
		font-size:105%;
		line-height:1.3;

		}
	h1	{
		font-family:helvetica;
		color:#404040;
		font-size:140%;
		}
	h2	{
		font-family:helvetica;
		color:#404040;
		font-size:130%;
		}
	.picture	{
				font:14px/1.4em Helvetica, sans-serif;
				color:#787878;
				padding:0px;
				margin-left:auto;
				margin-right:auto;
				}
	.picture img	{
					border:2.5px solid #C8C8C8;
					vertical-align:middle;
					margin-bottom:3px;
					margin-left:auto;
					margin-right:auto;
					}
	.right	{
			margin:7.8em 1.5em 4.5em 6.5em;
			float:right;
			}
</style>


<h2>Description</h2>
<p>
The degree of marine–freshwater divergence is quantified using a Cluster Separation Score statistic which measures average pairwise genetic distance between marine and freshwater groups after accounting for the average distance within each of the groups (see Methods and Supplementary Equation 1 in <a href="https://doi.org/10.1038/nature10944">Jones <i>et al.</i> 2012</a> for more details). Intervals represented in this track have experienced strong (FDR &lt; 0.05 or FDR &lt; 0.02) parallel divergence between marine and freshwater populations. This is best illustrated by the pattern at the Ectodysplasin (EDA) locus, a key gene previously shown to control repeated evolution of armor plate differences between marine and freshwater sticklebacks (<a href="https://doi.org/10.1126/science.1107239">Colosimo <i>et al.</i> 2005</a>).  Here, the alternative ancient alleles have been repeatedly selected and driven to high frequency in marine and freshwater habitats. Analyses were performed on genomic windows of 2500bp, stepping by 500bp.
</p>

<h2>Methods</h2>
<p>
Data were calculated from Illumina 36bp single-end short-read whole genome sequencing of 21 individual threespine stickleback fish from globally sampled (Pacific and Atlantic basins) marine and freshwater populations -- originally in gasAcu1 coordinates, which have been lifted to gasAcu1-4. For a detailed description of methods refer to <a href="https://doi.org/10.1038/nature10944">Jones <i>et al.</i> 2012</a>.

</p>

<h2>Credits</h2>
<p style=font-size:95%>
Felicity C Jones<sup>&#42;,1</sup>, Manfred G Grabherr<sup>&#42;,2,3</sup>, Yingguang Frank Chan<sup>&#42;,1,&#134;</sup>, Pamela Russell<sup>&#42;,2,</sup>, Evan Mauceli<sup>2</sup>, Jeremy Johnson<sup>2</sup>, Ross Swofford<sup>2</sup>, Mono Pirun<sup>2,&#167;</sup>, Michael C Zody<sup>2</sup>, Simon White<sup>4</sup>, Ewan Birney<sup>5</sup>, Stephen Searle<sup>4</sup>, Jeremy Schmutz<sup>6</sup>, Jane Grimwood<sup>6</sup>, Mark C Dickson<sup>6</sup>, Richard M Myers<sup>6</sup>, Craig T Miller<sup>1,&#135;</sup>, Brian R Summers<sup>1</sup>, Anne K Knecht<sup>1</sup>, Shannon D Brady<sup>1</sup>, Haili Zhang<sup>1</sup>, Alex A Pollen<sup>1</sup>, Timothy Howes<sup>1</sup>, Chris Amemiya<sup>7</sup>, Broad Institute Genome Sequencing Platform and Whole Genome Assembly Team<sup>2</sup>, Eric S Lander<sup>2</sup>, Federica Di Palma<sup>2</sup>, Kerstin Lindblad-Toh<sup>&#35;,2,3</sup>, and David M Kingsley<sup>&#35;,1,8</sup>
</p>

<p></p>
<i><blockquote>
<p style="font-size:80%">
<sup>1</sup> Department of Developmental Biology, Beckman Center B300, Stanford University School of Medicine, Stanford CA 94305, USA</br>
<sup>2</sup> Broad Institute of MIT and Harvard, 7 Cambridge Center, Cambridge MA 02142, USA</br>
<sup>3</sup> Science for Life Laboratory Uppsala, Department of Medical Biochemistry and Microbiology, Uppsala University, Uppsala 751 23, Sweden</br>
<sup>4</sup> Wellcome Trust Sanger Institute, Hinxton CB10 1SA, UK</br>
<sup>5</sup> European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK</br>
<sup>6</sup> HudsonAlpha Institute for Biotechnology, 601 Genome Way, Huntsville, AL 35806, USA</br>
<sup>7</sup> Department of Molecular Genetics, Benaroya Research Institute at Virginia Mason, 1201 Ninth Avenue, Seattle WA 98101, USA</br>
<sup>8</sup> Howard Hughes Medical Institute, Stanford University, Stanford, CA 94305, USA</br>
</p>

<p></p>
<p style="font-size:80%">
<br>Current address:</br>
<sup>&#134;</sup> Max Planck Institute for Evolutionary Biology, August-Thienemann-Str. 2, Pl&ouml;n 24306, Germany</br>
<sup>&#167;</sup> Bioinformatics Core, Zuckerman Research Center, New York, NY 10065</br>
<sup>&#135;</sup> Department of Molecular &amp; Cell Biology, 142 LSA &#35;3200, University of California, Berkeley, CA 94720, USA</br>
<sup>&#42;</sup> these authors contributed equally to this work</br>
<sup>&#35;</sup> co-corresponding authors.</br>
</i></p>
</blockquote>

<h2>References</h2>
<p>
Jones <i>et al.</i> (2012) <i>The genomic basis of adaptive evolution in threespine sticklebacks.</i> <a href="https://doi.org/10.1038/nature10944">Nature, 484: 55–61.</a></p>

<p>Colosimo <i>et al.</i> (2005) <i>Widespread Parallel Evolution in Sticklebacks by Repeated Fixation of Ectodysplasin Alleles.</i> <a href="https://doi.org/10.1126/science.1107239">Science, 307 (5717): 1928-1933.</i></a>
</p>

<h2>Contact</h2>
  
<p>Please email <a href="mailto:kingsley@stanford.edu">David Kingsley</a>, <a href="mailto:kingsley_lab@mailman.stanford.edu">the Kingsley Lab</a>, or <a href="mailto:krishna.Veeramah@stonybrook.edu">Krishna Veeramah</a> with questions about the hub or the most recently published associated manuscript (<a href="https://doi.org/10.1126/sciadv.abg5285">Roberts Kingman et al. 2021</a>).</p>
 
