<!DOCTYPE html>
<html>

<p>The assembly <code>gasAcu1-4</code> is a new version of the threespine stickleback reference genome. It only minorly differs from the <a href="=https://doi.org/10.1093/jhered/esx058">2017 Hi-C guided assembly generated by Peichel, Sullivan, Liachko, and White</a> to improve the subtelomeric <em>Pitx1</em> locus and the mitochondrial genome.</p>

<p>The subtelomeric region of <code>chrVII</code> is of great biological interest due to its well-documented role in controlling pelvic spine development (<a href="https://doi.org/10.1038/nature02415">Shapiro et al. 2004</a>). However, due to its highly repetitive nature, it is extremely difficult to assemble and many important sequences, including the key gene <em>Pitx1</em>, are missing entirely from existing genome assemblies, while other sequences in the region are scattered in small unassembled scaffolds. We address this issue by including the sequence from Salmon River BAC clones (Genbank GU130435) (<a href="https://doi.org/10.1126/science.1182213">Chan et al. 2010</a>) as <code>chrP</code> and removing overlapping fragmented sequences from <code>chrUn</code> and the end of <code>chrVII</code>. We note that <code>chrP</code> is derived from a marine population, while the rest of the genome is from a freshwater population (Bear Paw Lake), so all analyses concerning this chromosome must be interpreted as such.</p>

<p>The mitochondrial genome was previously split into two fragments buried within <code>chrUn</code>, while a separate mitochondrial genome sequence from Northern Japan was added as <code>chrM</code>. We corrected these issues by removing the duplicated Bear Paw Lake mitochondrial genome from <code>chrUn</code> and using it to replace the exogenous <code>chrM</code> sequence, resulting in a single copy of the mitochondrial genome derived entirely from Bear Paw Lake.</p>

<p>Aside from these changes, the <a href="=https://doi.org/10.1093/jhered/esx058">Peichel et al. 2017</a> reference genome was not modified in any other ways.</p>

<p><b><u><code>Liftover</code> chains.</b></u> <code>LiftOver</code> (<a href="https://doi.org/10.1093/nar/gkj144">Hinrichs et al. 2006</a>) chains (<a href="https://doi.org/10.1073/pnas.1932072100">Kent et al. 2003</a>) between <code>gasAcu1</code> and <code>gasAcu1-4</code> were generated from <code>BLAT</code> (<a href="https://doi.org/10.1101/gr.229202">Kent 2002</a>) alignments using the script <code><a href="https://github.com/ENCODE-DCC/kentUtils/blob/master/src/hg/utils/automation/doSameSpeciesLiftOver.pl">doSameSpeciesLiftOver.pl</a></code> as well as <code><a href="https://github.com/ucscGenomeBrowser/kent">Kent Utilities</a></code> following the procedure described here: <a href="http://genomewiki.ucsc.edu/index.php/DoSameSpeciesLiftOver.pl">http://genomewiki.ucsc.edu/index.php/DoSameSpeciesLiftOver.pl</a>.</p>

<p><b><u>Gene Annotation.</b></u> Ensembl (<a href="https://doi.org/10.1093/database/baw093">Aken et al. 2016</a>) release 94 gene annotations for <code>gasAcu1</code> were downloaded via the UCSC Genome Browser and lifted to <code>gasAcu1-4</code> using all default <code>LiftOver</code> parameters.</p>

<p><b><u>Major Quantitative Trait Loci.</b></u> In order to annotate our new genome build with quantitative trait loci (QTL), we began with all QTLs from the comprehensive 2017 review of all reported stickleback QTLs by <a href="https://doi.org/10.1098/rstb.2015.0486">Marques and Peichel</a>. We first filtered out results from stickleback species other than Gasterosteus aculeatus, and then for each reported interval lifted the coordinates from <code>gasAcu1</code> to <code>gasAcu1-4</code>. Finally, to create a set of well-bounded major QTL, we required each QTL to have PVE &gt; 20 and have a reported confidence interval &lt; 5Mb. All markers on chrP were assigned to the end of <code>chrVII</code>, where <em>Pitx1</em> physically resides. No filtering was performed on account of trait similarity within a cross or multiple crosses analyzing the same trait. This yielded 108 major QTLs. Some loci contain more overlapping QTL reports than visualized in <b>Fig. 1G</b> of <a href="https://doi.org/10.1126/sciadv.abg5285">Roberts Kingman et al. 2021</a>.</b></p>

<p><b><u>Acknowledgements.</b></u> We thank A. Hinrichs, H. Clawson, K. Smith, and D. Karolchik for contribution to the UCSC Stickleback Genome Browser annotations for <code>gasAcu1</code>, which were lifted to <code>gasAcu1-4</code> for this study.</p>
  
</html>
