<!DOCTYPE html>
<html>
<h1>Welcome to the <a href="https://kingsley.stanford.edu">Kingsley Lab</a> <code>gasAcu1-4</code> assembly hub</h1>
<p>The assembly <code>gasAcu1-4</code> is a new version of the threespine stickleback reference genome. It only minorly differs from the <a href="=https://doi.org/10.1093/jhered/esx058">2017 Hi-C guided assembly generated by Peichel, Sullivan, Liachko, and White</a> to improve the subtelomeric <em>Pitx1</em> locus and the mitochondrial genome.</p>

<p>The subtelomeric region of <code>chrVII</code> is of great biological interest due to its well-documented role in controlling pelvic spine development (<a href="https://doi.org/10.1038/nature02415">Shapiro et al. 2004</a>). However, due to its highly repetitive nature, it is extremely difficult to assemble and many important sequences, including the key gene <em>Pitx1</em>, are missing entirely from existing genome assemblies, while other sequences in the region are scattered in small unassembled scaffolds. We address this issue by including the sequence from Salmon River BAC clones (Genbank <a href="https://www.ncbi.nlm.nih.gov/nuccore/GU130435">GU130435</a>) (<a href="https://doi.org/10.1126/science.1182213">Chan et al. 2010</a>) as <code>chrP</code> and removing overlapping fragmented sequences from <code>chrUn</code> and the end of <code>chrVII</code>. We note that <code>chrP</code> is derived from a marine population, while the rest of the genome is from a freshwater population (Bear Paw [BEPA] Lake), so all analyses concerning this chromosome must be interpreted as such.</p>

<p>The mitochondrial genome was previously split into two fragments buried within <code>chrUn</code>, while a separate mitochondrial genome sequence from Northern Japan was added as <code>chrM</code>. We corrected these issues by removing the duplicated Bear Paw Lake mitochondrial genome from <code>chrUn</code> and using it to replace the exogenous <code>chrM</code> sequence, resulting in a single copy of the mitochondrial genome derived entirely from Bear Paw Lake.</p>

<p>Aside from these changes, the <a href="https://doi.org/10.1093/jhered/esx058">Peichel et al. 2017</a> reference genome was not modified in any other ways.</p>

<h3>Methods</h3>
  
<p>Please see the individual track description pages for details on the data within each track, or consult <a href="https://doi.org/10.1126/sciadv.abg5285">Roberts Kingman et al. 2021</a>.</p>
  
<!--<p><b><u><code>Liftover</code> chains.</b></u> <code>LiftOver</code> (<a href="https://doi.org/10.1093/nar/gkj144">Hinrichs et al. 2006</a>) chains (<a href="https://doi.org/10.1073/pnas.1932072100">Kent et al. 2003</a>) between <code>gasAcu1</code> and <code>gasAcu1-4</code> were generated from <code>BLAT</code> (<a href="https://doi.org/10.1101/gr.229202">Kent 2002</a>) alignments using the script <code><a href="https://github.com/ENCODE-DCC/kentUtils/blob/master/src/hg/utils/automation/doSameSpeciesLiftOver.pl">doSameSpeciesLiftOver.pl</a></code> as well as <code><a href="https://github.com/ucscGenomeBrowser/kent">Kent Utilities</a></code> following the procedure described here: <a href="http://genomewiki.ucsc.edu/index.php/DoSameSpeciesLiftOver.pl">http://genomewiki.ucsc.edu/index.php/DoSameSpeciesLiftOver.pl</a>.</p>

<p><b><u>Gene Annotation.</b></u> Ensembl (<a href="https://doi.org/10.1093/database/baw093">Aken et al. 2016</a>) release 94 gene annotations for <code>gasAcu1</code> were downloaded via the UCSC Genome Browser and lifted to <code>gasAcu1-4</code> using all default <code>LiftOver</code> parameters.</p>

<p><b><u>Major Quantitative Trait Loci.</b></u> In order to annotate our new genome build with quantitative trait loci (QTL), we began with all QTLs from the comprehensive 2017 review of all reported stickleback QTLs by <a href="https://doi.org/10.1098/rstb.2015.0486">Marques and Peichel</a>. We first filtered out results from stickleback species other than Gasterosteus aculeatus, and then for each reported interval lifted the coordinates from <code>gasAcu1</code> to <code>gasAcu1-4</code>. Finally, to create a set of well-bounded major QTL, we required each QTL to have PVE &gt; 20 and have a reported confidence interval &lt; 5Mb. All markers on chrP were assigned to the end of <code>chrVII</code>, where <em>Pitx1</em> physically resides. No filtering was performed on account of trait similarity within a cross or multiple crosses analyzing the same trait. This yielded 108 major QTLs. Some loci contain more overlapping QTL reports than visualized in <b>Fig. 1G</b> of <a href="https://doi.org/10.1126/sciadv.abg5285">Roberts Kingman et al. 2021</a>.</b></p> -->

<h3>Acknowledgements</h3>

<p>We thank A. Hinrichs, H. Clawson, K. Smith, and D. Karolchik for contribution to the UCSC Stickleback Genome Browser annotations for <code>gasAcu1</code>, which were lifted to <code>gasAcu1-4</code> for this study.</p>

<h3>Data Availability</h3>
  
<p>A copy of the files underlying this assembly hub can also be downloaded from <a href="https://figshare.com/projects/kingsleyAssemblyHub/162634">FigShare project # 162634</a>.</p>

<h3>Contact</h3>
  
<p>Please email <a href="mailto:kingsley@stanford.edu">David Kingsley</a>, <a href="mailto:kingsley_lab@mailman.stanford.edu">the Kingsley Lab</a>, or <a href="mailto:krishna.Veeramah@stonybrook.edu">Krishna Veeramah</a> with questions about the hub or the associated manuscript (<a href="https://doi.org/10.1126/sciadv.abg5285">Roberts Kingman et al. 2021</a>).</p>
  

-----

<h4>What is a genome browser and how do I make it work?</h4>

<p>We live at a remarkable time when the complete DNA sequence of many living organisms is being decoded. To facilitate the study of genome-wide information, several groups have developed powerful software tools that make it possible to search, visualize, interpret, and download the genetic instruction set of sequenced species. For basic information about how the browser works, see <a href="https://doi.org/10.1002/0471142905.hg1806s71">Karolchik et al. 2011</a>, or click on the Help menu available in the blue bar at the top of the browser window.</p>

<h4>How do I find a particular gene or marker in the stickleback genome?</h4>

<p>You can enter a gene symbol, or keyword, or chromosome coordinate position in the search box at the top of the browser. Examples: ectodysplasin, EDA, chrIV:12,800,220-12,810,446</p>

<h4>What are those complicated looking names found in the gene prediction tracks?</h4>

<p>Ensembl has a very useful computational pipeline for predicting likely genes and gene products in large-scale genome assemblies (<a href="https://doi.org/10.1101/gr.1858004">Curwen et al. 2004</a>). The predictions for sticklebacks take into account sequence motifs and information from &gt;300,000 sequenced ESTs from a variety of stickleback cDNA libraries (<a href="https://doi.org/10.1163/1568539042948150">Kingsley et al. 2004</a>). Each of the resulting gene predictions are assigned unique IDs made up of the following letters and numbers: ENS for Ensembl; GAC for Gasterosteus aculeatus: the single letter G or T or P to denote either a predicted gene, transcript or protein; and then a string of numbers unique to each prediction (e.g.: ENSGACT00000026917). The Ensembl designations work as search terms in the browser. All Ensembl genes can also be viewed and intersected with other data using the "Ensembl genes" track found beneath the browser window.</p>

<h4>How can I find informative genetic markers for studying genomic regions of interest?</h4>

<!-- <p>Stanford microsatellite markers ("Stnxxx") and SNP genotyping chip markers ("rsxxxxxxxxx") have been used in many previous QTL mapping and population genetic experiments (for microsatellites see Peichel et al. 2001; for SNP genotyping chips see Jones et al 2012a). These markers can be searched by name in the search box. They can be visualized in the genome by toggling the "Stn Markers" or "Stanford_GenotypingChips_SNPs" button shown under "Marker Tracks" beneath the browser window.</p> -->

<p>If you want to design new microsatellite markers to tag a specific gene or genomic region, you can visualize promising locations where simple repeat sequences have been identified in the reference stickleback genome. If not already visible, display the "Simple Repeats" and "Microsatellites" tracks by clicking the "Repeats" section beneath the genome browser window. DNA primers made to unique sequences flanking di- and tri-nucleotide repeats have a high likelihood of revealing size polymorphism in other individuals and populations (<a href="http://doi.org/10.1038/414901a">Peichel et al. 2001</a>).</p>

<p>Finally, you can also use browser to view the positions of millions of genomic variants predicted from re-sequencing data in 227 natural stickleback populations from the Pacific and Atlantic ocean basins (<a href="https://doi.org/10.1038/nature10944">Jones et al. 2012b</a> and <a href="https://doi.org/10.1126/sciadv.abg5285">Roberts Kingman et al. 2021</a>).</p>

<p>Toggle the "227 Genomes Variants" track beneath the browser window to full view. The 227 visual genotype tracks that appear show the predicted value of these polymorphic loci in the 227 different marine and freshwater individuals that were sequenced. If you are zoomed in far enough in any genomic region, you can hover your pointer above the position of any colored variant position and see the exact genome position, as well as the value of the corresponding position in the reference Bear Paw (BEPA) genome, and the alternative allele detected in reads from other genomes.</p>

<h4>Can I download the genome-wide variant information that comes from large-scale stickleback re-sequencing?</h4>

<p>Single nucleotide polymorphisms (SNPs) and other short sequence variants identified by <a href="https://doi.org/10.1126/sciadv.abg5285">Roberts Kingman et al.</a> are contained in the track
called “227 Genomes Variants." You can use the Table Browser tool in the browser to download all variants in particular regions, as well as the corresponding genotype calls in different populations. Click on the “Table Browser" heading found under the "Tools" section in the blue banner at the top of the browser window. In the new table browser page that opens up, select clade: “kingsleyAssemblyHub", genome: “G. aculeatus (BEPA with SALR chrP)", assembly: “gasAcu1-4", group: “All Tracks", track: “227 Genomes Variants", table: “hub_3964941_227genomes_variants", and enter the coordinates of your favorite region (or choose the entire gasAcu1-4 genome). Note that by also clicking on the intersection button in the table browser, you can intersect the sequence variant information with many other interesting tracks available in the browser. For example, to get a list of the subset of variants occurring within predicted genes, start a table search beginning with the "227 Genomes Variants" track as described above, then click the intersect button, and then choose track: "Ensembl genes". Additional controls in the table browser make it possible to restrict your search to particular fields within a track, get summary statistics for your search, and choose different formats for exporting the data. A copy of the Variant Call File (VCF) underlying this track is available at <a href="https://doi.org/10.6084/m9.figshare.22325620">FigShare</a>.</p>

<h4>How do I find genomic regions that show repeated differentiation between sequenced marine and freshwater populations?</h4>

<p>In 2005, <a href="https://doi.org/10.1126/science.1107239">Colosimo et al.</a> showed that repeated evolution of armor plate changes in many different freshwater sticklebacks occurred via an ancient shared haplotype at the Ectodysplasin locus, which encodes a key developmental signaling molecule. The repeated use of ancient haplotypes gives rise to a distinctive pattern of allele sharing: all fish that share the same armor phenotypes also share similar distinctive sequences at the key locus controlling the armor trait, a pattern that is dramatically different than the phylogeographic patterns seen at other neutral loci (<a href="https://doi.org/10.1126/science.1107239">Colosimo et al. 2005</a>). <a href="https://doi.org/10.1038/nature10944">Jones et al. 2012b</a> looked for this distinctive pattern throughout the genome using two different computational methods. You can visualize the other repeatedly used regions found by marine-freshwater Cluster Separation Scores (CSS) by turning on the corresponding tracks listed under "Jones_et_al_2012" controls beneath the browser window. Click on the underlined text above any track button to get a brief summary of the type of information presented in the track, and consult the Jones et al. 2012b paper for detailed information about the CSS methods. The default window in the browser opens to the region surrounding the prototypical Ectodysplasin gene, and illustrates the characteristic sequence patterns now being used to recover many other loci that also underlie repeated evolution in natural environments.</p>

<p>An Excel table listing key regions recovered by the CSS approach is available as part of the <a href="https://static-content.springer.com/esm/art%3A10.1038%2Fnature10944/MediaObjects/41586_2012_BFnature10944_MOESM89_ESM.xls">supplementary information</a> in <a href="https://doi.org/10.1038/nature10944">Jones et al. 2012b</a>. These 81 regions can be viewed in bed format by turning on the track "CSS_0.02" in the "Jones_et_al_2012" Track Group at the bottom of the page. You can also do your own custom searches and downloads of CSS data using the "Table Browser" tool. Navigate to “Tools&gt;Table Browser" in the blue banner at the top of the browser. In the new Table Browser window that opens up, select group: "Jones_et_al_2012" and set track: to any of the particular r CSS analysis you want to look at in greater detail. You can intersect these tracks with any other tracks in the browser, get summary statistics for your search, and download data in a variety of other formats by choosing the appropriate buttons at the bottom of the Table Browser.</p>

<h4>What is chromosome "Un"?</h4>

<p>Genome sequencing projects are jigsaw puzzles with millions of pieces. The primary sequence data are short sequence reads made up of strings of A, C, G, and Ts. Many of these sequences overlap with each other, and can be aligned to produce longer continuous strings of sequence, called "contigs". Adjacent "contigs" can be bridged into larger "scaffolds", based on bridging clones whose end sequences map to different contigs. Finally, the larger "scaffolds" can be roughly positioned on chromosomes by following the inheritance pattern of genetic sequences in families. However, the process of connecting, lining up, and ordering all pieces is still incomplete. Most of the large sequence scaffolds have been tied to previously mapped chromosome linkage groups. In some of these cases, there was only a single informative genetic marker within a scaffold, or no recombination between internal markers within the scaffold in the small genetic mapping cross used. In these cases, the scaffold has been correctly assigned to a particular linkage group, but the orientation of the scaffold within that linkage group was arbitrary and may be changed by further mapping. In addition, many of the smaller scaffolds in the genome assembly did not contain any mapped genetic markers. These scaffolds have been concatenated together into an artificial "Unmapped" chromosome, so that they can still be searched and analyzed in the browser. As sequence and mapping information continues to grow in the future, more and more Unmapped scaffolds will be linked to sequences on other chromosomes and the large scale order and orientation of all scaffolds will continue to improve.</p>

<h4>Is anything missing from the stickleback genome?</h4>

<p>Yes!! Sequence reads in highly repetitive regions are difficult to align and assemble. For example, the <em>Pitx1</em> gene maps near a chromosome end, and the reads in this highly repetitive sub-telomeric region failed to incorporate with the rest of the stickleback assembly (<a href="https://doi.org/10.1126/science.1182213">Chan et al. 2010</a>). We think that this problem affects a relatively small number of genes, since more than 97% of cloned RNAs from stickleback tissues (ESTs), DO have corresponding genes in the reference assembly. If you don't find a stickleback ortholog of your favorite gene in the reference assembly, you can try searching against raw reads in the <a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastn&amp;BLAST_PROGRAMS=megaBlast&amp;PAGE_TYPE=BlastSearch&amp;BLAST_SPEC=SRA&amp;SHOW_DEFAULTS=on">trace archives</a> as well, setting the database to "Gasterosteus aculeatus".</p> 

<p>Although <a href="https://doi.org/10.1126/sciadv.abg5285">Roberts Kingman et al.</a> predict millions of genome wide sequence variants, many additional polymorphisms have undoubtedly been missed because of the requirement that all predicted variants be supported by a minimum number of reads. This criterion helps minimize false positives in high throughput sequencing data, and works well for finding regions of shared divergence between many different marine and freshwater populations. However, the same method will under-recover variants that are unique to single individuals or populations.</p>

<p>Finally, any automated annotation of genes and transcripts across an entire genome is likely to have many local prediction errors, including: undetected genes, incorrect start or termination sites, missed exons, concatenation of exons from separate genes into single predictions, poor recovery of small genes or non-coding RNAs, etc. Comparison of automated gene predictions with the actual expressed sequences identified in stickleback tissues or other organisms may help sort out some of these prediction errors<!-- (see "Stickleback ESTs" and "Other mRNAs" tracks under mRNA and EST tracks beneath the browser window)-->. However, the gold standard for confirming interesting observations will always be actual experiments. The genome browser can serve as a convenient starting point for exploring massive amounts of genetic information about the rapidly evolving stickleback species complex. We hope you will find it useful as you get you started on your own questions and research!</p>


</html>
