<h3>Please see <a href="https://doi.org/10.1126/sciadv.abg5285">Roberts Kingman et al., Sci Adv, 2021</a> for complete Materials and Methods information.</h3>

<h3>GENERAL SAMPLING METHODS AND BACKGROUND FOR CONTEMPORARY EVOLUTION LAKES</h3>

Annual samples were made from populations in three Alaskan lakes (Loberg, Cheney and Scout) to provide an evolutionary time series from founding to study rapid genomic evolution (<a href="https://www.adfg.alaska.gov/index.cfm?adfg=fishingSportLakeData.main&StockingAreaID=2">https://www.adfg.alaska.gov/index.cfm?adfg=fishingSportLakeData.main&StockingAreaID=2</a> for lake information, <a href="https://doi.org/10.1126/sciadv.abg5285">Figure</a> S1). These populations had been founded recently by anadromous (sea-run) threespine stickleback, and genetically determined phenotypic traits were evolving rapidly in all of them. Sample details are in <a href="https://doi.org/10.1126/sciadv.abg5285">Table</a> S1. The anadromous threespine stickleback population that breeds in Rabbit Slough, about 1.25 km from Loberg Lake, or a closely related population from an adjacent tributary in the same drainage probably founded the Loberg Lake population naturally. Mike A. Bell introduced anadromous Rabbit Slough adults into Cheney and Scout lakes in 2009 and 2011, respectively. Thus, all three lake populations that we used to study contemporary genomic evolution were derived from the same or closely related anadromous threespine stickleback populations.

<h3>CMH ANALYSIS OF CONTEMPORARY EVOLUTION SAMPLES</h3>

<p>In order to identify SNPs that had shown a significant change in allele frequency in our contemporary time-series data, we applied a modified Cochran–Mantel–Haenszel (CMH) test. CMH tests are commonly used in evolve and resequencing (E&R) experiments when comparing replicate Pool-Seq samples from two different time points (versus a Fisher’s exact test which only examines a single replicate) and show good power compared to many other tests (108). The modified CMH test, in similar vein to the N<span style='font-size:8.0pt;position:relative;top:2.0pt'>e</span> analysis described above, additionally accounts for the overdispersion caused by sampling and sequencing, while also controlling for the expected neutral drift between two time points, which better controls the resulting p-value distribution. This method does not account for the effect of uneven pooling of samples. However, in modelling drift in our modified CMH test, we utilize for each lake the harmonic mean of N<span style='font-size:8.0pt;position:relative;top:2.0pt'>e</span> estimates based on each successive pair of time points (Loberg = 250, Cheney = 150, Scout = 120). As these N<span style='font-size:8.0pt;position:relative;top:2.0pt'>e</span> estimates also did not account for possible uneven pooling, their values may in fact somewhat reflect the added overdispersion caused by it.</p>

<p>We applied this test to five combinations of lakes: all three lakes together (CH+SC+LB), Cheney and Scout together (CH+SC, this would reflect significant changes occurring within the earliest stage of freshwater adaptation), as well as each lake on its own (LB, CH, SC). We note that the first two combinations do not strictly represent replications as usually applied in evolve and resequence experiments, as each lake is sampled over different lengths of time, which the original code does not allow. We rewrote the code in python, allowing a different level of drift (N<span style='font-size:8.0pt;position:relative;top:2.0pt'>e</span>) for each individual lake when calculating the test statistic, as well as different time intervals. For Cheney and Scout we compared the earliest Pool-Seq time point (CH2009 and SC2011) to the latest time point (CH2017 and SC2017). In the case of Loberg for which we do not know the founding population precisely, we utilize RS2009 as a proxy for the starting population and LB2017 as the end point. As described above, Rabbit Slough is the most likely source of anadromous fish after Loberg was poisoned. </p>

<p>For each lake combination a modified CMH test statistic and associated p-value was calculated for any SNP identified in the established population set that for each Pool-Seq population had coverage between one-half and two times the mean coverage, and that was variable in at least one Pool-Seq population considered. The number of SNPs used ranged between 3,023,631 (CH+SC+LB) and 3,682,352 (LB) across the five combinations. We used a Bonferroni correction to identify significant SNPs at a 5% threshold. </p>

<p>For CH+SC+LB the number of significant SNPs is greatest on chrIV (n=46,215) but also noticeably prominent on chrVII (n=31,083) and XXI (n=28,299), while chrI, chrXI, chrXII and chrXX are somewhat more moderate. Barely any (<40) significant SNPs are found on chrVI, chrXV and chrXVIII (<a href="https://doi.org/10.1126/sciadv.abg5285">Figure</a> S33). LB analyzed on its own largely reiterates the relative number of significant SNPs as CH+SC+LB, presumably reflective of it being the most evolved freshwater population with regards to time (<a href="https://doi.org/10.1126/sciadv.abg5285">Figure</a> S34). It is noticeable that SC and CH analyzed separately show somewhat distinctive patterns, with chrIV also dominant when analyzing SC (<a href="https://doi.org/10.1126/sciadv.abg5285">Figure</a> S35) but chrXI is more prominent for CH (<a href="https://doi.org/10.1126/sciadv.abg5285">Figure</a> S36). Combining the signal for CH+SC shows the importance of chrIV in early freshwater adaptation, consistent with the genome-wide F<span style='font-size:8.0pt;position:relative;top:2.0pt'>ST</span> results (<a href="https://doi.org/10.1126/sciadv.abg5285">Figure</a> S37). As would be expected, there are more significant SNPs when utilizing multiple populations (e.g. CH+SC+LB) or populations with a long time-series (e.g. Loberg) in the CMH test.</p>

<p>Examining the distribution of CMH p-values across the genome using both physical and genetic distance (based on our Rabbit Slough-based recombination map), we see that our significant p-values cluster into clear peaks, while also noting that for chromosomes with large numbers of significant SNPs, using genetic distance tends to reduce the span of these peaks (i.e. they are much closer in genetic space compared to physical space) (<a href="https://doi.org/10.1126/sciadv.abg5285">Figure</a> S38-S42).</p>

<h3>TEMPOPEAK IDENTIFICATION</h3>

<p>We followed the same general <a href="https://doi.org/10.1126/sciadv.abg5285">EcoPeak identification methodology</a> in order to define TempoPeaks from our CMH p-values. However, we restricted our peaks to those called using a Bonferroni correction to determine the minimum p-value peak threshold, which showed a good overlap with long term peaks. We found using a false discovery rate to be too permissive compared to calling EcoPeaks. We produced a set of sensitive and specific TempoPeaks for each of the five lake combinations (CH+SC+LB, CH+SC, LB, CH, SC). Sensitive TempoPeaks were based on a 5% Bonferroni-corrected p-value threshold merging SNP and window-defined peaks. Specific TempoPeaks were based on a 1% p-value threshold only considering window-defined peaks (we found that including SNP-defined peaks in this case resulted in 10% fewer TempoPeaks but with less overlap with known EcoPeaks, suggesting they resulted in the removal of true positives). A separate set of peaks was also constructed using physical and genetic distance to define windows and peaks. Based on our overall genetic map, the approximate recombination rate per generation per base is 4e-6, and thus the window size and sliding window size of 2500bp and 50bp is equivalent to 0.01cM and 0.0002cM. </p>

<p>Unsurprisingly, the most TempoPeaks are identified using all three contemporary lakes together, where we will have maximum power, or using Loberg, where the most time has passed since the founding of the population, while the least number of TempoPeaks are found when only examining Scout, where the least time has passed (<a href="https://doi.org/10.1126/sciadv.abg5285">Table</a> S5). No significant specific TempoPeaks were found for Scout at all, though there were observable peaks in CMH scores that line up with significant TempoPeaks identified in other lake combinations (i.e. there is insufficient power to call them significant beyond the background of the rest of the genome at this stage of the lake’s evolution) (<a href="https://doi.org/10.1126/sciadv.abg5285">Figure</a> S43-S52). </p>

<p>As reflected in the distribution of SNPs with significant CMH p values and the genome-wide increase in F<span style='font-size:8.0pt;position:relative;top:2.0pt'>ST</span> over time, a large proportion of peaks are found on chrIV across all lake combinations, though other chromosomes appear to also be almost equally prominent when only looking at Loberg, suggesting that chrIV is driving the earliest stages of freshwater adaptation as reflected by Cheney and Scout, but that other chromosomes "catch up" later on (<a href="https://doi.org/10.1126/sciadv.abg5285">Figure</a> S53-<a href="https://doi.org/10.1126/sciadv.abg5285">Figure</a> S57). </p>

<p>Using genetic distance always results in fewer peaks than the equivalent test with physical distance, by ~20-40% in the case of sensitive TempoPeaks and ~60% for specific TempoPeaks (<a href="https://doi.org/10.1126/sciadv.abg5285">Table</a> S5). It appears that some regions of the genome that appear as multiple distinct peaks in physical distance are likely single peaks when taking into account variation in the recombination rate, which is also reflected in the total size of TempoPeaks being of similar magnitude using genetic versus physical distance (2.8% versus 3.8% of the genome for CH+SC+LB for specific TempoPeaks, <a href="https://doi.org/10.1126/sciadv.abg5285">Table</a> S6) and with the genetic distance TempoPeaks generally being large despite the reduced number (<a href="https://doi.org/10.1126/sciadv.abg5285">Table</a> S7). TempoPeak sizes vary considerably. When using genetic distance and looking at CH+SC+LB specific TempoPeaks using genetic distance, the median TempoPeak size is ~31kb, but with a 5th and 95% percentile that ranges from ~2.5 to 275kb, and the largest peak being ~1.5Mb long (chrIV: 20862645-22379208). </p>

<p>Almost all specific TempoPeaks identified in the contemporary evolution samples overlap with sensitive EcoPeaks in the geographic analysis (<a href="https://doi.org/10.1126/sciadv.abg5285">Table</a> S8), strongly suggesting that these regions reflect true positives that exhibit consistent recurrent and rapid evolution in threespine stickleback. There is much less overlap for the sensitive TempoPeaks identified in CH+SC+LB and LB (on the order of 40%), presumably either due to the evolution of specific local peaks or false positives, though there is a noticeable improvement when looking at CH and SC. The presence of considerable false positives amongst the sensitive sets is also likely reflected in the total amount of the genome found in significant TempoPeaks, where for both CH+SC+LB and LB ~ 20% of the genome is found in sensitive TempoPeaks, while this value is <3% in the case of specific TempoPeaks (<a href="https://doi.org/10.1126/sciadv.abg5285">Table</a> S6). Even the most lenient set of global EcoPeaks and regions from <a href="https://doi.org/10.1038/nature10944">Jones et al. 2012</a>, in contrast, overlap only 96/344 and 47/344 specific TempoPeaks, respectively (<a href="https://doi.org/10.1126/sciadv.abg5285">Table</a> S9, S10). Although these still constitute significant statistical enrichments (<a href="https://doi.org/10.1126/sciadv.abg5285">Table</a> S9, S10), the low proportion of overlapped TempoPeaks indicate that globally shared standing variation represents only a fraction of the genomic regions under selection in the contemporary evolution populations. </p>

<p>When examining specific TempoPeaks defined by genetic distance (the most conservative conditions), we found 12 TempoPeaks in CH+SC that were absent in corresponding specific LB analysis. These could be rapidly evolving loci that were not selected for during the accidental colonization of Loberg, perhaps because they were absent in the initial founders (<a href="https://doi.org/10.1126/sciadv.abg5285">Figure</a> S58). However, we also note that these are present in the LB sensitive TempoPeak set, and so may reflect limited statistical power due to slower evolution. </p>


<h3>Contact</h3>
  
<p>Please email <a href="mailto:kingsley@stanford.edu">David Kingsley</a>, <a href="mailto:kingsley_lab@mailman.stanford.edu">the Kingsley Lab</a>, or <a href="mailto:krishna.Veeramah@stonybrook.edu">Krishna Veeramah</a> with questions about the hub or the associated manuscript (<a href="https://doi.org/10.1126/sciadv.abg5285">Roberts Kingman et al. 2021</a>).</p>
